test/mocha/001-schemas.js

```diff
@@ -83,6 +91,268 @@ describe('bedrock-validation', function() {
         should.not.exist(result);
         err.name.should.equal('UnknownSchema');
       });
+    it('should return middleware with a ValidationError due to invalid body',
+      function() {
```

Comment:
We should pass `done` in here as a callback and then call it in `next` below. The reason for doing this is that `next` could be called on a different turn of the event loop. It's true that it presently *isn't*, but better to model good behavior in our tests if we can.

```suggestion
      function(done) {
```